zenodo/modules/records/models.py

```diff
@@ -214,7 +220,7 @@ def get_types(cls):
     def get_subtypes(cls, type_):
         """Get object type value."""
         cls._load_data()
-        return cls.subtypes[type_]
+        return cls.subtypes.get(type_, [])
```

Comment:
```suggestion
        return cls.subtypes.get(type_)
```

I'm a bit hesitant overall about changing the behavior of this. Before if a `type_` was invalid an error would be raised